hermes-client/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     testCompile group: 'org.spockframework', name: 'spock-core', version: versions.spock
     testCompile group: 'com.github.tomakehurst', name: 'wiremock', version: versions.wiremock
     testCompile group: 'javax.servlet', name: 'javax.servlet-api', version: '3.1.0'
+    testCompile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.2.0'
```

Comment:
wiremock has dependency on json-path and by default that version cause some issues on our classpath